content/en/guides/employee-engagement.md

```diff
@@ -0,0 +1,179 @@
+
+When you have been pointed by your company to get into an Open Source Project and you are a new person who wants to contribute to the project the first thing you want to do is get a feeling of how the project is functioning. The first things to do are: read the contributing document, if there is one and if there is no contributing document you could find out how the project works and based on your findings your first contribution could be writing the Contributing document.
+
+Also check the email mailing lists (normally those are published on the project's site), discussion forums, project repository (repo), and what kind of pull requests and bug/issue reports there are. Some projects use Slack, IRC, Discourse, Discord, Mattermost, etc. channels, so make sure that you subscribe to those. Whichever channel is used, it is good to introduce yourself: who you are, which company you represent, etc. It is good to be transparent that you are in the project on behalf of the company. Once you have introduced yourself, a good practice is to ask for suggestions on where to get started and what kind of contributions are good for newcomers. Remember you are working for a community and it is community needs that come first (see Principle 2).
+
+Some projects have weekly or monthly meetings so if the time zones allow, it is good to participate in those meetings too to get a feeling of the project and the people participating in it.
+
+When you have questions, it is very likely that someone has had the same questions before, so it is again a good practice to search the project's discussion channels / mailing list emails first to see if your question has already been answered. If your question is a new one, go ahead and ask it. Again, if you think that your question is something that should have the answer in the project's readme, contributing or other project document, please make sure that you add the info there and make your contribution.
+
+By starting with listening you will learn how the project works, you will also learn about what is not written down, but which is reflected in culture. It allows you and your organization to map the informal network and norms of the project.
+
+And if you are completely new to open source this is a good way to learn how pull requests, forking/branching, review and more work. Open source projects are very welcoming if you approach them respectfully, are willing to listen, and act on what you have learned.
+
+### Suggested tactics for starting with listening
+
+* Read the README file
+* Read the CONTRIBUTING file
+* Look for the main communication channel and have a look at the archives to get an impression how the community communicates
+* Have a look at closed issues and pull requests to learn how contributions work
+* Look for recordings of presentations of the project at conferences to learn more about the project and the people who speak about it
+
```

Comment:
- Go to where the project is. This can include you needing to use communication and source control platforms that you might have opinions about but as a new contributor you will need to do the work to join their community.
- If you have a question, first search issues/pull requests/message boards/FAQs first to see if others have already answered your question. If not, use the proper pathways to raise your question.

Reply:
Incorporated in #397.